packages/design-system/src/scalars/components/sidebar/sidebar.stories.tsx

```diff
@@ -106,8 +106,8 @@ const meta: Meta<typeof Sidebar> = {
       </div>
     ),
     enableMacros: 4,
-    onActiveNodeChange: (nodeId) => {
-      console.log("onActiveNodeChange", nodeId);
+    onActiveNodeChange: (node) => {
+      console.log("onActiveNodeChange", node);
```

Comment:
Is this console.log necessary?

Reply:
Note that this is in a story, it is for testing on the storybook

Reply:
nice, thanks for the feedback